### PR TITLE
WIP: welcome page w/demo

### PR DESCRIPTION
**Pull request by ShawnMilo**
Demo [video](http://youtu.be/SiD65S37-lI).

### CODE REVIEW

**Comment by ShawnMilo** [comment]
The pep8/pylint stuff has the same issues as before with `no-member`, and the migrations file has its usual issues. If anyone knows the `.rc` file magic to make pep8 skip migrations please let me know.

**Comment by Ferdi** [comment]
No ui in learningobjects please.
- Sent from my phone

**Comment by ShawnMilo** [comment]
Update: Disregard. The designer will probably manipulate the forms enough that it doesn't make sense to have vanilla ones sitting in the app dir. I'll move the templates. When you say "no ui," does that just apply to the templates, or anything else?

**Comment by ShawnMilo** [comment]
Latest push: `templates` folder and HTML templates removed from `learningobjects` app and moved to new `ui` app.

**Comment by carsongee** [comment]
I mentioned it in chat last week, but the migration fix to pep8 is:

``` ini
pep8ignore =
    */migrations/* ALL
```

in pytest.ini

File: ui/templates/learningobjects/create_repo.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<h1>Create repository</h1>
```
Comment (Ferdi) [inline ui/templates/learningobjects/create_repo.html]: Can we have all template files be under ui/templates/ ?

Comment (ShawnMilo) [inline ui/templates/learningobjects/create_repo.html]: Yep, no problem. This is another thing I'll do after the listing page branch is merged into this.

**Comment by ShawnMilo** [comment]
Updating to WIP so I can stop messing with this until #21 is merged to prevent extra rebasing.

**Pull request closed without merging**